### ISSUE TITLE
Ability to disable authentication

### ISSUE THREAD

**Issue by valodiadeseynes** [othmar52/slimpd#59]
Yes I know, it's not the most secure thing in the world but for people like me who run slimpd on their LAN for non-techy users, I would like them to just open their browser without having to remember their username/password or having to chose the guest account.

**Comment by othmar52** [comment]
i agree - until the authentication is not implemented properly it may be the best to remove this feature at all.
the idea was to have a functional remember user and/or password (which can be empty) functionality. so theoretically there should be no need to remember any username or password in LAN environments.

But i havn't finished fixing all the authentication bugs.
